content/hi/docs/tutorials/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: ट्यूटोरियल
+main_menu: true
+no_list: true
+weight: 60
+content_type: concept
+---
+
+<!-- overview -->
+
+प्रलेखन के इस खंड में ट्यूटोरियल हैं।
+ट्यूटोरियल दिखाता है कि किसी एकल कार्य से बड़े [लक्ष्य](/hi/docs/tasks/) को कैसे पूरा किया जाए। 
+आमतौर पर एक ट्यूटोरियल में कई खंड होते हैं,
+जिनमें से प्रत्येक में चरणों के क्रम होते हैं।
+प्रत्येक ट्यूटोरियल से परिचित होने से पहले, हम आपको 
+[मानकीकृत शब्दावली](/hi/docs/reference/glossary/) पृष्ट को बुकमार्क करने की सलाह देते हैं। 
+<!-- body -->
+
+## मूलभूत
+
+* [मूलभूत कुबरनेट्स](/hi/docs/tutorials/kubernetes-basics/) एक गहन इंटरैक्टिव ट्यूटोरियल है जो आपको कुबेरनेट्स प्रणाली को समझने और कुबेरनेट्स की कुछ बुनियादी सुविधाओं को आज़माने में मदद करता है।
+
+* [Introduction to Kubernetes (edX)](https://www.edx.org/course/introduction-kubernetes-linuxfoundationx-lfs158x#)
+
+* [Hello Minikube](/hi/docs/tutorials/hello-minikube/)
+
+## विन्यास
+
+* [उदाहरण: Java माइक्रोसर्विस का विन्यास करना](/hi/docs/tutorials/configuration/configure-java-microservice/)
+
+* [कॉन्फिग मैप का उपयोग करके Redis का विन्यास करना](/hi/docs/tutorials/configuration/configure-redis-using-configmap/)
+
+## स्टेटलेस एप्लीकेशन
+
+* [किसी क्लस्टर में किसी एप्लिकेशन तक पहुंचने के लिए बाहरी आईपी पते को उजागर करना](/hi/docs/tutorials/stateless-application/expose-external-ip-address/)
+
+* [उदाहरण: रेडिस के साथ PHP गेस्टबुक एप्लिकेशन को उपयोग करना](/hi/docs/tutorials/stateless-application/guestbook/)
+
+## स्टेटफुल एप्लीकेशन
+
+* [स्टेटफुलसेट मूलभूत](/hi/docs/tutorials/stateful-application/basic-stateful-set/)
+
+* [उदाहरण: दृढ़ वॉल्यूम के साथ वर्डप्रेस और MySQL](/hi/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/)
+
+* [उदाहरण: कैसेंड्रा को स्टेटफुल सेट के साथ उपयोग करना](/hi/docs/tutorials/stateful-application/cassandra/)
+
+* [एक CP वितरित प्रणाली, ज़ूकीपर चलाना](/hi/docs/tutorials/stateful-application/zookeeper/)
```

Comment:
Avoid linking to pages that don't already exist (see earlier suggestions)

Reply:
Oops, I will take care